docs/guides/development/cli.md

```diff
@@ -261,21 +261,23 @@ Although a lot of Pack management can be done through the CLI, there are still s
 
 It's possible to start development of a Pack in the Pack Studio web editor and later migrate to using the CLI. For example, to take advantage of an NPM library, which isn't possible in the web editor.
 
-To migrate, follow all of the steps described in the [Getting started tutorial][quickstart_cli] up until the point of running `npx coda create`. Instead, manually create a file in the same directory as the pack definition called `.coda-pack.json` with the following content:
+To migrate, first make sure you have the [required software](#requirements) installed. Then create a new directory for the Pack, [install the SDK](#install), and [register an API token](#register). Finally run `coda clone` passing in the URL of the Pack.
 
-```json
-{
-  "packId": 123456
-}
+```sh
+npx coda clone "https://coda.io/p/123456"
 ```
 
-Replace `123456` with the ID of your existing Pack. You can find this ID in the Pack Studio URL:
+This will initialize the directory with the recommended file structure, download your existing Pack code into `pack.ts`, and create a `.coda-pack.json` file that links them together.
 
-```
-https://coda.io/p/123456
-```
+!!! info "Link only"
```

Comment:
Curious about this syntax ("!!!") also

Reply:
This uses the Admonition extension for Python markdown:

https://python-markdown.github.io/extensions/admonition/

It renders the content as a styled aside. See the link in the previous comment for a preview.